### PR TITLE
update branch to main

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -51,7 +51,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1w
```
Comment (cachedout) [inline .ci/jobs/defaults.yml:54]: This seems unrelated to the purpose of this PR. If this was intended, perhaps move it to a different PR?

**Review by bmorelli25 — APPROVED** [review]
Awesome! Thanks for updating the docs!

File: .ci/jobs/defaults.yml [hunk 2]
```diff
@@ -51,7 +51,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
```
Comment (v1v) [inline .ci/jobs/defaults.yml:53]:
```suggestion
        wipe-workspace: 'True'
        periodic-folder-trigger: 1w
```

**Review by cachedout — APPROVED** [review]

**Review by Mpdreamz — APPROVED** [review]
https://github.com/elastic/ecs-dotnet/blob/315218167ebf79e19fa9ef09a99bf48675ed74ef/build/scripts/Targets.fs#L134

Needs to be updated to but is unrelated to this repo moving to main. It's an effect of ecs-logging moving.

**Comment by v1v** [comment]
> ecs-dotnet/build/scripts/Targets.fs

Thanks for the heads up, I'll commit those changes here

**Review by Mpdreamz — APPROVED** [review]
Thanks @v1v !
